### PR TITLE
6151 update feature highlights

### PR DESCRIPTION
**Pull request by wyli**
part of #6151

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: docs/source/whatsnew_1_2.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# What's new in 1.2 🎉🎉
+
+- Auto3DSeg enhancements and benchmarks
+- nnUNet integration
+- TensorRT optimized networks
```
Comment (mingxin-zheng) [inline docs/source/whatsnew_1_2.md:5]: How about "TensorRT-optimized networks"?

File: docs/source/whatsnew_1_2.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+## nnUNet integration
+The integration introduces a new class, `nnUNetV2Runner`, which leverages Python APIs to facilitate model training, validation,
+and ensemble, thereby simplifying the data conversion process for users.
+Benchmarking results from various public datasets confirm that nnUNetV2Runner performs as expected.
+Users are required to prepare a data list and create an `input.yaml` file to install and use the system.
+The framework also allows automatic execution of the entire nnU-Net pipeline, from model training to ensemble,
+with options to specify the number of epochs. Users can access APIs for training, dataset conversion, data preprocessing, and other components.
+Please check out [the tutorials](https://github.com/Project-MONAI/tutorials/tree/main/nnunet) for more details.
+
+## TensorRT optimized networks
```
Comment (mingxin-zheng) [inline docs/source/whatsnew_1_2.md:28]: Same comment as above. Thanks!

File: docs/source/whatsnew_1_2.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+via https://github.com/Project-MONAI/MetricsReloaded. Binary and categorical metrics computing modules are included in this release,
+using MetricsReloaded as the backend. [Example scripts](https://github.com/Project-MONAI/tutorials/tree/main/modules/metrics_reloaded) are made available to demonstrate the usage.
+
+
+## Bundle workflow APIs
+`BundleWorkflow` abstracts the typical workflows (such as training, evaluation, and inference) with three main interfaces:
+`initialize`, `run`, and `finalize`, applications use these APIs to execute a bundle.
+It unifies the required properties and optional properties for the workflows, downstream applications
+can invoke the methods instead of parsing configs with keys.
+In this release, `ConfigWorkflow` class is also created for JSON and YAML config-based bundle workflows for improved pythonic usability.
```
Comment (mingxin-zheng) [inline docs/source/whatsnew_1_2.md:51]: The `p` in `pythonic` should be uppercase?

File: docs/source/whatsnew_1_2.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+
+## Modular patch inference
+In patch inference, patches are extracted from the image, the inference is run on those patches, and outputs are merged
+to construct the result image corresponding to the input image. Although depending on the task, model, and computational/memory resources,
+the exact implementations of a patch inference may vary, the overall process of splitting, running inference and merging the results remains the same.
+In this release, we have created a modular design for patch inference, where it defines the overall process while abstracts away the specific
```
Comment (mingxin-zheng) [inline docs/source/whatsnew_1_2.md:58]: Two minor suggestions:

- `where it` -> `which`
- `while abstracts` -> `while abstracting`

File: docs/source/whatsnew_1_2.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+`initialize`, `run`, and `finalize`, applications use these APIs to execute a bundle.
+It unifies the required properties and optional properties for the workflows, downstream applications
+can invoke the methods instead of parsing configs with keys.
+In this release, `ConfigWorkflow` class is also created for JSON and YAML config-based bundle workflows for improved pythonic usability.
+
+
+## Modular patch inference
+In patch inference, patches are extracted from the image, the inference is run on those patches, and outputs are merged
+to construct the result image corresponding to the input image. Although depending on the task, model, and computational/memory resources,
+the exact implementations of a patch inference may vary, the overall process of splitting, running inference and merging the results remains the same.
```
Comment (mingxin-zheng) [inline docs/source/whatsnew_1_2.md:57]: Add a comma: inference**,** and

File: docs/source/whatsnew_1_2.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+Users are required to prepare a data list and create an `input.yaml` file to install and use the system.
+The framework also allows automatic execution of the entire nnU-Net pipeline, from model training to ensemble,
+with options to specify the number of epochs. Users can access APIs for training, dataset conversion, data preprocessing, and other components.
+Please check out [the tutorials](https://github.com/Project-MONAI/tutorials/tree/main/nnunet) for more details.
+
+## TensorRT optimized networks
+[NVIDIA TensorRT](https://developer.nvidia.com/tensorrt) is an SDK for high-performance deep learning inference,
+includes a deep learning inference optimizer and runtime that delivers low latency and high throughput for inference applications.
+It can accelerate the deep learning model forward computation on the NVIDIA GPU.
+In this release, the `trt_export` API to export the TensorRT engine-based TorchsScript model has been integrated into the MONAI bundle.
```
Comment (KumoLiu) [inline docs/source/whatsnew_1_2.md:32]:
```suggestion
In this release, the `trt_export` API to export the TensorRT engine-based TorchScript model has been integrated into the MONAI bundle.
```

File: docs/source/whatsnew_1_2.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+
+## MetricsReloaded integration
+MetricsReloaded - a new recommendation framework for biomedical image analysis validation - is released publicly
+via https://github.com/Project-MONAI/MetricsReloaded. Binary and categorical metrics computing modules are included in this release,
+using MetricsReloaded as the backend. [Example scripts](https://github.com/Project-MONAI/tutorials/tree/main/modules/metrics_reloaded) are made available to demonstrate the usage.
+
+
+## Bundle workflow APIs
+`BundleWorkflow` abstracts the typical workflows (such as training, evaluation, and inference) with three main interfaces:
```
Comment (Nic-Ma) [inline docs/source/whatsnew_1_2.md:47]: suggestion:
"typical workflows" -> "typical workflows of a bundle".

Thanks.

File: docs/source/whatsnew_1_2.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+`initialize`, `run`, and `finalize`, applications use these APIs to execute a bundle.
```
Comment (Nic-Ma) [inline docs/source/whatsnew_1_2.md:48]: "execute a bundle" -> "execute a bundle workflow".

Thanks.

File: docs/source/whatsnew_1_2.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+It unifies the required properties and optional properties for the workflows, downstream applications
+can invoke the methods instead of parsing configs with keys.
```
Comment (Nic-Ma) [inline docs/source/whatsnew_1_2.md:50]: Suggestion:
"methods" -> "components".

Thanks.

**Review by KumoLiu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**